includes/Modules/Idea_Hub.php

```diff
@@ -613,7 +630,8 @@ protected function setup_settings() {
 	 * @return Asset[] List of Asset objects.
 	 */
 	protected function setup_assets() {
-		$base_url = $this->context->url( 'dist/assets/' );
+		$transients = $this->transients;
```

Comment:
Same here, no need to assign it to a variable, just use `$this->transients->...` directly.

Reply:
Please see my question above ☝️